webrtc.html

```diff
@@ -1561,6 +1561,53 @@
             the <a href="#dom-peerconnection-ice-gathering-state"><code>RTCPeerConnection</code>
             ice gathering state</a> changes.
           </dd>
+
+          <dt>attribute EventHandler onunassignedmedia</dt>
+
+          <dd>
+            <p>
+              This event handler, of type <code><a>unassignedmedia</a></code>,
+              MUST be fired by objects implementing
+              the <code><a>RTCPeerConnection</a></code> interface.  It is fired
+              when authenticated media packets arrive from the remote peer that
+              are not assigned to a <code><a>MediaStreamTrack</a></code>.
+              Unauthenticated packets are always discarded without generating an
+              event.
```

Comment:
What do you mean by unauthenticated. There are two cases here ... one is a ranodm packet with where SRTP integrity fails, and one where a valid DTLS session is set up and the packet is valid SRTP encrypted with keys set up in DTLS but there is not yet a answer with a fingerprint that for that DTLS session. We don't want to drop those - the whole point is to get notified about them.